cluster/watchdog.go

```diff
@@ -121,6 +121,9 @@ func (w *Watchdog) rescheduleContainers(e *Engine) {
 			}
 		}
 
+		// Clear out the network configs, since we cannot create a
+		// container with multiple networks.
+		c.Config.NetworkingConfig.EndpointsConfig = nil
```

Comment:
The created container would not have the original's `NetworkingConfig.EndpointsConfig`, isn't it?